### PR TITLE
Add more supports for cursor pagination

### PR DESCRIPTION
**Pull request by hotwatermorning**
* Slack added cursor pagination options for some APIs on Aug 8th.
    * https://api.slack.com/docs/pagination
* To support cursor pagination for the APIs, update `slack-api-ref` and fix some methods in `web/api/endpoints/*.rb`.

### CODE REVIEW

**Comment by dblock** [comment]
We typically include the API changes here, so you might want to break up these PRs if it's easier.

**Comment by hotwatermorning** [comment]
> We typically include the API changes here in CHANGELOG, please add.

Let me see. Should the description of this changes in `CHANGELOG.md` be like this?

```
[#215](https://github.com/slack-ruby/slack-ruby-client/pull/215): Add cursor pagination support for `groups_list`, `files_info`, `mpim_list`, `reactions_list`, `stars_list` - [@hotwatermorning](https://github.com/hotwatermorning).
```

Excuse me, I had forgot to do `rubocop --auto-gen-config`.
Now I did the command, added changes of `.rubocop_todo.yml` into 3701595, and pushed.

**Comment by hotwatermorning** [comment]
> Aren't the modified files for pagination auto-generated? They should not be altered by hand.

I see. The modified files are auto-generated by `rake slack:api:update`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ### 0.11.2 (Next)
 
 * Your contribution here.
+* [#215](https://github.com/slack-ruby/slack-ruby-client/pull/215): Add more supports for cursor pagination - [@hotwatermorning](https://github.com/hotwatermorning).
```
Comment (dblock) [inline CHANGELOG.md]: Please update this with all the changes in separate items by looking at the diff. For example "Added `apps_permissions_users_list`" or "`files_info`, `...` now support pagination". Thx.

**Comment by dblock** [comment]
This looks good. Update the changelog with all the changes and it's good to go.

**Comment by hotwatermorning** [comment]
Thank you for your kind advice.

I fixed the changelog and pushed. How about this?

**Comment by dblock** [comment]
Good, merged. Thanks.

**Pull request merged**